src/components/menu/menu.scss

```diff
@@ -35,12 +35,12 @@ $menu-item-background-color: #ebebeb;
         }
 
         &:not(.limel-portal--fixed) {
+            z-index: -999999;
```

Comment:
This doesn't affect the z-index of the dropdown itself, only the z-index of the `limel-portal` element, which the dropdown is positioned after. We need this to have a negative z-index to not end up covering the trigger and making the trigger impossible to click.